test/testcafe/spec/ChallengeOktaVerifySSOExtensionView_spec.js

```diff
@@ -103,11 +103,11 @@ test
     await t.expect(getStateHandleFromSessionStorage()).eql(null);
   });
 
-// TODO: OKTA-557948 Implement custom biometrics error and enable this test
-test.meta('v3', false)
+test.meta('v3', true)
```

Comment:
Since the entire spec is enabled for v3, you can just remove the disable meta flag at the test level and it will automatically be included for v3.
```suggestion
test
```